---
title: "PhD Defence  üßë‚Äçüéì"
comments: false
showBreadcrumbs: false
showReadingTime: false
showTOC: false
---

Hello to all,  
as you might now for the last couple of years I have been preparing my PhD and the time has finally come for it to end. I will soon be defending my PhD called **"From sequences to knowledge, improving and learning from sequence alignments"**. This defence will be **in English**.  

It will be happening on **December 2nd** at **2:PM**, at the [Jussieu campus](https://goo.gl/maps/fV5BX99xdRsy6wNy7) of Sorbonne universit√©, in the **Durand** auditorium of the **Esclangon** building *(cf. map below)*. You can download a calendar event [here](/files/defence.ics).  
If you cannot come in person, no problem! The defence will also be attendable remotely with Microsoft Teams (link [here](https://teams.microsoft.com/l/meetup-join/19%3ameeting_YjQ2NDliYjktMzQ5OS00MDc5LWIyYzItNmFkNjdhMmRmMGM3%40thread.v2/0?context=%7b%22Tid%22%3a%22096815dc-d9eb-4bc3-a5a3-53c77e7d34e2%22%2c%22Oid%22%3a%22efed87df-6c6d-40e8-8d70-14ad4e775309%22%7d))  

It will be followed by some drinks and some food at the university and then by a party at a nearby bar *(details T.B.A)*. If you cannot make it to the defence I hope you can at least make it to the bar!

![Map of the campus](/images/amphi_durand.jpg#center)

### So that I can organize this event, please fill out [this form](https://framaforms.org/soutenance-de-luc-lucs-defence-1667484172), also available at [the bottom of the page](#form)

## Jury
The defence will be done in front of my PhD Jury, the composition of which is given below:

{{< markdowntable >}}

|                                                                           |                     |                |
| ------------------------------------------------------------------------- | ------------------- | -------------- |
| [Brona Brejova](http://compbio.fmph.uniba.sk/~bbrejova/)                  | Associate Professor | Reviewer       |
| [Macha Nikolski](https://dept-info.labri.fr/~macha/)                      | Group Leader        | Reviewer       |
| [√âlodie Laine](http://www.lcqb.upmc.fr/laine/Home.html)                   | Associate Professor | Examinator     |
| [Olivier Gascuel](https://isyeb.mnhn.fr/fr/annuaire/olivier-gascuel-7496) | Research Director   | Examinator     |
| [Jean-Philippe Vert](https://members.cbio.mines-paristech.fr/~jvert/)     | Research Director   | Examinator     |
| [Paul Medvedev](https://medvedevgroup.com/principal-investigator/)        | Associate Professor | Invited Member |
| [Rayan Chikhi](http://rayan.chikhi.name/)                                 | Group Leader        | PhD Advisor    |

{{< /markdowntable >}}

## Schedule

The tentative schedule of the defence is as follows:

{{< markdowntable >}}

|   |   |   |
|---|---|---|
| **2** PM    | **2:45** PM         | PhD Presentation |
| **2:45** PM | **3:30** ~ **4** PM | Jury Questions |
| **4** PM    | **4:30** PM         | Jury Deliberation |
| **4:30** PM | **4:45** PM         | Jury Verdict  |
| **4:45** PM | **6:30** PM         | Food and Drinks  |
| **7** PM    | **?**               | Party üéâ |

{{< /markdowntable >}}


## My thesis

If you wish to read my manuscript it is available as a website ([thesis.lucblassel.com](https://thesis.lucblassel.com)), or as a [pdf document](https://thesis.lucblassel.com/_main.pdf). Good Luck!

### Abstract 

In this thesis we study two important problems in computational biology, one pertaining to primary analysis of sequencing data, and the second pertaining to secondary analysis of sequences to obtain biological insights using machine-learning. Sequence alignment is one of the most powerful and important tools in the field of computational biology. Read alignment is often the first step in many analyses like structural variant detection, genome assembly or variant calling. Long read sequencing technologies have improved the quality of results across all these analyses. They remain, however, plagued by sequencing errors and pose algorithmic challenges to alignment. A prevalent technique to reduce the detrimental effects of these errors is homopolymer compression, which targets the most common type of long-read sequencing error. We present a more general framework than homopolymer compression, which we call mapping-friendly sequence reductions (MSR). We then show that some of these MSRs improve the accuracy of read alignments across whole human, *drosophila* and *E. coli* genomes. Improvements in sequence alignment methods are crucial for downstream analyses. For instance, multiple sequence alignments are indispensable when studying resistance in viruses. With the ever growing quantity of annotated, high-quality multiple sequence alignments it has become possible and useful to study drug resistance in viruses with machine learning methods. We used a very large multiple sequence alignment of British HIV sequences to train multiple classifiers to discriminate between treatment-naive and treatment-experienced sequences. By studying important classifier features we identified resistance-associated mutations. We then removed known drug resistance associated signal from the data before training, keeping classifying power, and identified 6 novel resistance associated mutations. Further study indicated that these were most likely accessory in nature and linked to known resistance mutations.

## Form

{{< rawhtml >}}
<iframe src="https://framaforms.org/soutenance-de-luc-lucs-defence-1667484172" width="100%" height="800" border="0">
</iframe>
{{< /rawhtml >}}